sys/dev/drm/panfrost/panfrost_mmu.c

```diff
@@ -646,9 +628,23 @@ panfrost_mmu_init(struct panfrost_softc *sc)
 static void
 panfrost_mmu_release_ctx(struct panfrost_mmu *mmu)
 {
+	struct panfrost_softc *sc;
+
+	sc = mmu->sc;
+
+	iommu_pmap_remove_pages(&mmu->p);
+	iommu_pmap_release(&mmu->p);
+
+	mtx_lock_spin(&sc->as_mtx);
```

Comment:
Oof, why is this using spin mutexes everywhere?  Is that due to drm-subtree?

Reply:
mtx_lock_spin is only used by panfrost code, core DRM bits use SX locks I think

Reply:
I'll check. Possibly some code is running in the interrupt time